src/naip_cnn/data.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import geopandas as gpd
+import numpy as np
+import tensorflow as tf
+
+from naip_cnn.sampling import _parse_pixel_array
+
+
+def load_parquet_dataset(
+    path: str, target_names: tuple[str], **kwargs
+) -> tf.data.Dataset:
+    """Load a Tensorflow Dataset from a Parquet file.
+
+    Parameters
+    ----------
+    path : str
+        Path to the Parquet file.
+    target_names : tuple[str]
+        Names of the target columns to load.
+    **kwargs
+        Keyword arguments to pass to `naip_cnn.sampling._parse_pixel_array`.
+    """
+    df = gpd.read_parquet(path)
+    images, labels = list(
+        zip(
+            *(
+                (_parse_pixel_array(row, **kwargs), row.loc[list(target_names)].values)
+                for _, row in df.iterrows()
+            )
+        )
+    )
+    images = np.stack(images).astype(np.float32) / 255
+    labels = np.stack(labels).astype(np.float32) / 100
```

Comment:
Yes. My current plan is to keep the package focused for this use case to avoid getting bogged down trying to maximize flexibility, so I'm thinking uint8 NAIP will probably be the only supported imagery. Also I'm not sure if this was clear, but I was planning to just keep the package as an internal tool to support the notebooks, rather than publishing it separately. What do you think - should we aim for something more flexible?

You're definitely right that the scaler for labels should be more general though, since we may be applying this to other metrics. Actually, I don't *think* the magnitude of the label affects the model (I just scaled it here because I was used to comparing MSEs at this magnitude), so we can probably ditch the scaling altogether.

Reply:
All sounds good to me (private scope and not scaling the labels).